e2e/main.py

```diff
@@ -422,35 +559,47 @@ def call_bulk(bulks, *, batch_size):
         assert (
             queued_tez > 10_000_000_000
         ), "queued tez in liquidation auction was not greater than Constants.max_lot_size which is required for this test"
-
+        print(f"Auction queue currently has {queued_tez} mutez")
         call_endpoint(checker, "touch", None)
 
         # Now that there is an auction started in a descending state, we can select a queued slice
         # and be confident that we can cancel it before it gets added to another auction.
         # To successfully do this, we also need to either move the index price such that the
         # cancellation is warranted or deposit extra collateral to the burrow. Here we do the latter.
-        for queued_leaf_ptr, leaf in auction_avl_leaves(
-            checker,
-            checker.storage["deployment_state"]["sealed"]["liquidation_auctions"][
-                "queued_slices"
-            ](),
+        cancel_ops = []
+        for i, (queued_leaf_ptr, leaf) in enumerate(
+            auction_avl_leaves(
+                checker,
+                checker.storage["deployment_state"]["sealed"]["liquidation_auctions"][
+                    "queued_slices"
+                ](),
+            )
         ):
-            # Only retrieve the first queued slice ptr for efficiency
-            break
-        # Deposit a ton of tez to this burrow to ensure that it is no longer liquidatable
-        call_endpoint(
-            checker,
-            "deposit_tez",
-            leaf["leaf"]["value"]["contents"]["burrow"][1],
-            amount=100_000_000_000,
+            cancel_ops.append(
+                (
+                    checker.deposit_tez(
+                        leaf["leaf"]["value"]["contents"]["burrow"][1]
+                    ).with_amount(1_000_000_000),
+                    checker.cancel_liquidation_slice(queued_leaf_ptr),
+                )
+            )
+            if len(cancel_ops) >= 895:
+                break
```

Comment:
The queue has ~900 slices in it, and I was afraid to have to run this test suite too many times to check the limit :sweat_smile:. Happy to increase this though if you think it would be helpful.

Reply:
Ah, I see! No need, but maybe put this information in a comment here so that we don't trip over this in the future.